config/adapters_config.yaml

```diff
@@ -137,30 +137,32 @@ child_pathway_of:
   nodes: False
   edges: True
 
-ontology_terms:
+go_ontology_terms:
   adapter:
-    module: biocypher_metta.adapters.ontologies_adapter
-    cls: OntologyAdapter
+    module: biocypher_metta.adapters.gene_ontology_adapter
+    cls: GeneOntologyAdapter
     args:
       type: node
-      dry_run: False
+      dry_run: True
 
-  outdir: onotology
+  outdir: gene_onotology
   nodes: True
   edges: False
 
 onotology_relationships:
   adapter:
-    module: biocypher_metta.adapters.ontologies_adapter
-    cls: OntologyAdapter
+    module: biocypher_metta.adapters.gene_ontology_adapter
+    cls: GeneOntologyAdapter
     args:
+      label: subtype_of
       type: edge
-      dry_run: False
+      dry_run: True
```

Comment:
`dry_run` should be `False` in the main adapters config. It should be set `True` in the sample adapters config.